File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int i = 0;
        
        int n = nums.size();
        vector<int>ans;
        while(i<n){
            ans.push_back(nums[i]);
            while(i+1<n&&nums[i]==nums[i+1]){
                i++;
            }
            i++;
        }
        int a=ans.size();
        for(int in=0;in<a;in++){
            nums[in]=ans[in];
        }
        return a;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        arr=[]
        n=len(nums)
        j=0
        for i in range(0,n-1):
           if nums[i]!=nums[i+1]:
            arr.append(nums[i])
            j+=1

        arr.append(nums[-1])

        m=len(arr)
        for i in range(0,m):
            nums[i]=arr[i]

        return(len(arr))

        